docs/reference/builder.md

```diff
@@ -806,7 +806,7 @@ of whether or not the file has changed and the cache should be updated.
 > can only contain a URL based `ADD` instruction. You can also pass a
 > compressed archive through STDIN: (`docker build - < archive.tar.gz`),
 > the `Dockerfile` at the root of the archive and the rest of the
-> archive will get used at the context of the build.
+> archive will be used at the context of the build.
```

Comment:
oh, looks like there's another typo here;

"used at the context" should be "used _as_ the context"